spec/support/shared_examples/participant_actions_withdraw_support.rb

```diff
@@ -49,4 +49,15 @@
 
     expect(response).not_to be_successful
   end
+
+  context "with an invalid request" do
+    before { params[:data][:attributes][:reason] = "erroneous-reason" }
+
+    it "changes the training status of a participant to withdrawn" do
+      put url, params: params
+
+      expect(response).not_to be_successful
+      expect(parsed_response.dig("errors", 0, "detail")).to eq("Reason is not included in the list")
+    end
+  end
```

Comment:
"changes the training status of a participant to withdrawn"? Doesn't sound like what the test is doing